english/marshmallow.md

```diff
@@ -732,4 +732,5 @@ Instead of reading this entire story, you decide to go see the new star wars!
 
 You decide that $1.5 billion is too much to pass up, so you buy a [Powerball ticket](powerball/powerball.md).
 
-[Do you have my stapler?](stapler/stapler.md)
\ No newline at end of file
+[Do you have my stapler?](stapler/stapler.md)
+Join the new Milk Cafe where you can listen to Debussy all day and sip flavoured milks! [Milk Cafe this way!](milkcafe/milkcafe.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.